controller/src/beerocks/master/controller_ucc_listener.h

```diff
@@ -32,7 +32,8 @@ class controller_ucc_listener : public beerocks_ucc_listener {
                                   const std::string &dest_mac = std::string()) override;
     bool handle_dev_set_config(std::unordered_map<std::string, std::string> &params,
                                std::string &err_string) override;
-
+    bool handle_dev_get_param(std::unordered_map<std::string, std::string> &params,
+                              std::string &value) override;
```

Comment:
Using `std::string &value` instead of `std::string &err_string` for error is breaking the file convetion.
I get that you did it since it is not necessarily for error, and is using for the function positive output.
I would replace the return value to `std::string`, so in case it fails, it would return an empty string, and the err_string would be filled with the error description.
That way you won't have to define failure by the string "FAIL":

defintion:
```cpp
std::string handle_dev_get_param(std::unordered_map<std::string, std::string> &params,
                                 std::string &err_string)
```
use:
```cpp
auto param_value = handle_dev_get_param(params, err_string)
if (handle_dev_get_param(params, value).empty()) {
    reply_ucc(eWfaCaStatus::ERROR, err_string);
    break;
}
    
reply_ucc(eWfaCaStatus::COMPLETE, param_value);
```

(nitpick)

Reply:
Not needed - in case of failure, value will be set to the error reason.